### ISSUE TITLE
Switch to Tokens

### ISSUE THREAD

**Issue by mgibas** [gitcheese/gitcheese.ui#4]
_From @flyboarder on May 11, 2017 19:56_

I don't think many people will be happy with providing their banking information.

It would be better to integrate with, and allow using Stripe API tokens instead.

_Copied from original issue: mgibas/gitcheese.client#1_

**Comment by mgibas** [comment]
Good point - though we are not sending these data to gitcheese api (stripe api generates token for us instead):

[gc-user-managed-accounts-external-account-info.html](https://github.com/gitcheese/gitcheese.ui/blob/ce44623d0b8b64da787e18b0d213c06d553de8e0/src/gc-user/gc-user-managed-accounts-external-account-info.html#L24) and [gc-config.html](https://github.com/gitcheese/gitcheese.ui/blob/ce44623d0b8b64da787e18b0d213c06d553de8e0/src/gc-common/gc-config.html#L12)

so maybe communicating this in a clear way would be a good solution ?

**Comment by burn2delete** [comment]
I think the correct way would be to have uses acquire their token from the stripe dashboard.
